### ISSUE TITLE
Issue with @connectrpc/connect-query: Module not found: Error: Can't resolve './use-transport'

### ISSUE THREAD

**Issue by malinskibeniamin** [connectrpc/connect-query-es#298]
Hi there 👋 

Unfortunately after upgrading to use `@connectrpc/connect-query`, I found that my app stopped working. I am using a regular CRA setup, and it seems that despite the package exporting both CommonJS and ESM, some of the modules cannot be found, `use-transport`, `create-use-query-options` and `create-use-infinite-query-options` to name a few.

I tried using
```ts
import { useQuery } from '@connectrpc/connect-query/dist/cjs/use-query';
```
instead, but it doesn't seem to alleviate the issue.

Also it might be worth mentioning that npmjs has a [@connectrpc/connect-react-query](https://www.npmjs.com/package/@connectrpc/connect-react-query) release `1.0.0-rc.1` which might confuse some people, perhaps it might be worth removing it if [@connectrpc/connect-query](https://www.npmjs.com/package/@connectrpc/connect-query) should be used instead?

Please find the issue below:

```sh
ERROR in ./node_modules/@connectrpc/connect-query/dist/esm/use-mutation.js 16:0-47
Module not found: Error: Can't resolve './use-transport' in 'app/node_modules/@connectrpc/connect-query/dist/esm'
Did you mean 'use-transport.js'?

BREAKING CHANGE: The request './use-transport' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
ERROR in ./node_modules/@connectrpc/connect-query/dist/esm/use-query.js 15:0-67
Module not found: Error: Can't resolve './create-use-query-options' in 'app/node_modules/@connectrpc/connect-query/dist/esm'
Did you mean 'create-use-query-options.js'?

BREAKING CHANGE: The request './create-use-query-options' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
ERROR in ./node_modules/@connectrpc/connect-query/dist/esm/use-query.js 16:0-47
Module not found: Error: Can't resolve './use-transport' in 'app/node_modules/@connectrpc/connect-query/dist/esm'
Did you mean 'use-transport.js'?

BREAKING CHANGE: The request './use-transport' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
ERROR in ./node_modules/@connectrpc/connect-query/dist/esm/use-infinite-query.js 15:0-84
Module not found: Error: Can't resolve './create-use-infinite-query-options' in 'app/node_modules/@connectrpc/connect-query/dist/esm'
Did you mean 'create-use-infinite-query-options.js'?

BREAKING CHANGE: The request './create-use-infinite-query-options' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

```

**Comment by paul-sachs** [comment]
Sorry about that. Looks like JS module system is at it again. Gotta get around to fixing #186. In the mean time, I'll fix the import issue.

**Comment by malinskibeniamin** [comment]
Thanks a lot @paul-sachs for the quick fix, really appreciate it. Would it be possible to cut a new release with these changes? Thank you

**Comment by paul-sachs** [comment]
Yeah. I'm just reviewing a few more issues to get into the release. Will do a release today.

**Comment by paul-sachs** [comment]
Sorry for the delay. This bug has been fixed with release [v1.1.0](https://github.com/connectrpc/connect-query-es/releases/tag/v1.1.0).
